### DISCUSSION TITLE
stat-computing-r-python/part-gen-prog/07-prog-data

### DISCUSSION THREAD

**Discussion by giscus** [srvanderplas/stat-computing-r-python#21] [General]
# stat-computing-r-python/part-gen-prog/07-prog-data

https://srvanderplas.github.io/stat-computing-r-python/part-gen-prog/07-prog-data.html

**Comment by Sgraham12** [comment]
The NYC dogs data is unavailable, I'm not sure if this is a temporary problem or if the data has been moved to another place.

**Reply by srvanderplas** [reply]
It seems to work when I try it, so I'm guessing this was temporary.
